pkgs/development/python-modules/astropy/default.nix

```diff
@@ -39,6 +40,9 @@ buildPythonPackage rec {
     pytest
   '';
 
+  # 368 failed, 10889 passed, 978 skipped, 69 xfailed in 196.24s
```

Comment:
It would be good to explain why these tests fails, for later reference.

Reply:
I think there are a few reasons for the failure, however I'm not sure what they all are. That's why I'm unable to fix them.